test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -48,6 +48,20 @@ Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @(
     }
 }
 
+Describe "Validate that the Help function can Run in strict mode" -Tags @('CI') {
+
+    It "Help doesn't fail when strict mode is on." {
+
+        $help = & {
+            # run in nested scope to keep strict mode fromo affecting other tests.
```

Comment:
typo `from`, probably also remove the trailing punctuation which is inconsistent with general comments